concepts.md

```diff
@@ -255,7 +255,7 @@ A [Public Provisional Identity] can be generated from a [Secret Provisional Iden
   <dd>a user of the <em>application</em>, owning one or more <em>device</em>s</dd>
 
   <dt>user group</dt>
-  <dd>a group of <em>uer</em>s, the <em>group member</em>s, created and managed by <em>user</em>s. <em>Data</em> shared with a <em>user group</em> is accessible to all its <em>group member</em>s</dd>
+  <dd>a group of <em>user</em>s, the <em>group member</em>s, created and managed by <em>user</em>s. <em>Data</em> shared with a <em>user group</em> is accessible to all its <em>group member</em>s</dd> <-- this weird
```

Comment:
`<dd>a group of <em>user</em>s, created and managed by the <em>group members</em>s themselves. <em>Data</em> shared with a <em>user group</em> is accessible to all its <em>group member</em>s</dd>`